doc/source/reference/arrays.ndarray.rst

```diff
@@ -469,7 +469,7 @@ Comparison operators:
    ndarray.__eq__
    ndarray.__ne__
 
-Truth value of an array (:func:`bool()`):
+Truth value of an array (:class:`bool() <bool>`):
```

Comment:
Does removing the `()` also work here?

```suggestion
Truth value of an array (:func:`bool`):
```
My memory was that the `()` gets added automatically for `:func:` references, even if they actually point to classes. I might be wrong though.

Reply:
I tried it, but ``:func:`bool` `` role didn't work.

Reply:
Thanks for trying!